command/genesis/genesis.go

```diff
@@ -77,6 +77,13 @@ func setFlags(cmd *cobra.Command) {
 		"the burn contract block and address (format: <block>:<address>[:<burn destination>])",
 	)
 
+	cmd.Flags().Uint64Var(
+		&params.baseFee,
+		genesisBaseFeeFlag,
+		command.DefaultGenesisBaseFee,
+		"initial base fee for the chain in wei, Default BaseFee is 1 Gwei (London fork is disabled when baseFee is 0)",
```

Comment:
I didn't know that London fork is disabled if `baseFee` is set to 0. I thought that London fork is enabled only if `burn-contract` flag is provided. Should we add validation that `baseFee` must be greater than 0 in case London fork is enabled?